Project.toml

```diff
@@ -58,9 +58,9 @@ ImageShow = "0.3"
 JSON3 = "1"
 Lazy = "0.15"
 MIToS = "3"
-Makie = "0.19, 0.20, 0.21"
-Meshes = "0.43"
-MolecularGraph = "0.13, 0.14, 0.15, 0.16"
+Makie = "0.19, 0.20, 0.21, 0.22"
+Meshes = "0.43, 0.47, 0.48, 0.49, 0.50, 0.51, 0.52, 0.53"
+MolecularGraph = "0.13, 0.14, 0.15, 0.16, 0.17, 0.18"
```

Comment:
Does this clear up all the issues from CompatHelper as well?

Reply:
Yes, see the huge list of closed issues/PRs at the top.